### PR TITLE
NSC recipe

### PR DESCRIPTION
**Pull request by sw005320**
- An example of using [lhotse](https://github.com/lhotse-speech/lhotse)'s data preparation
- [ ] add some results
- [ ] tune the BPE size (5K would be a bit too large)
- [ ] change lang from en to en-sg

### CODE REVIEW

File: egs2/nsc/asr1/local/data.sh [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+    # Make a dev set
+    utils/subset_data_dir.sh --first data/train 4000 data/dev
+    n=$(($(wc -l < data/train/text) - 4000))
+    utils/subset_data_dir.sh --last data/train ${n} data/train_nodev
+
+    # Remove temp files
+    rm -f data/*.spk
+
+    # FIXME(sw005320): reco2dur is not properly handled in the current script
```
Comment (sw005320) [inline egs2/nsc/asr1/local/data.sh:63]: @kamo-naoyuki, I found that if we have `reco2dur` in the data directory, it fails in stage 4. Do you know why?

**Comment by pzelasko** [comment]
Nice! If you think that test set selection is reasonable, I can adjust it on Lhotse's side to return standard train/dev/test partitions (will simplify everybody's life).

**Comment by pzelasko** [comment]
Same goes for text normalization (I did not want to hard-code that in Lhotse without a second pair of eyes validating that this way of normalization is acceptable for a "standard recipe").

File: egs2/nsc/asr1/local/data.sh (hunk 1 above)
Comment (kamo-naoyuki) [reply egs2/nsc/asr1/local/data.sh:63]: what error?

Comment (sw005320) [reply egs2/nsc/asr1/local/data.sh:63]: I'll make a report when I make time.

**Comment by sw005320** [comment]
> Nice! If you think that this test set selection is reasonable, I can adjust it on Lhotse's side to return standard train/dev/test partitions (will simplify everybody's life).

I think this is reasonable.
If there is no official split, we can just standardize it by ourselves.

**Comment by pzelasko** [comment]
Out of curiosity, what’s the result you got? I looked for a number but couldn’t find it.

**Comment by sw005320** [comment]
I'll add some numbers and pre-trained models later.
The word error rate of the test set is around 31%.
